site/src/components/accordion.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Accordion
+description: The accordion component in the Atlas Design System
+template: standard
+# need design to create
```

Comment:
Remove entirely figma section, I doubt atlas's figma will be updated any time soon

Reply:
✅Updated in both md files